staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/status.go

```diff
@@ -68,7 +68,7 @@ func ErrorToAPIStatus(err error) *metav1.Status {
 		// by REST storage - these typically indicate programmer
 		// error by not using pkg/api/errors, or unexpected failure
 		// cases.
-		runtime.HandleError(fmt.Errorf("apiserver received an error that is not an metav1.Status: %#+v", err))
+		runtime.HandleError(fmt.Errorf("apiserver received an error that is not an metav1.Status: %#+v: %v", err, err))
```

Comment:
seems like the flag with %v has a special meaning  and will be pretty-printed if the type implements `GoStringer`

https://golang.org/pkg/fmt/#GoStringer

Reply:
should the errors returned by the server implement that interface?

Reply:
it's an error. `%v` will use the String func.

Reply:
do you still need to keep %#+v ?

Reply:
`%#+v` helps with debugging. This is most probably a programming error when it happens.